### PR TITLE
Added support for ULID

### PR DESCRIPTION
**Pull request by WengerK**
<h2>Change Log</h2>
<div id="change-log">
  <h4>Added</h4>
  <ul id="added">
  
<li><code>@ulid@</code> - matcher</li>
 
   
  </ul> 
  <h4>Fixed</h4>  
  <ul id="fixed">
    
  </ul>
  <h4>Changed</h4>
  <ul id="changed">
    
  </ul>  
  <h4>Removed</h4>
  <ul id="removed">
    
  </ul>
  <h4>Deprecated</h4>
  <ul id="deprecated">
    
  </ul>  
  <h4>Security</h4> 
  <ul id="security">
    
  </ul>     
</div>
<hr/>

<h2>Description</h2>

Added support for ULID (Universally Unique Lexicographically Sortable Identifier.

It uses a shared, pseudo-random source of entropy, seeded with the current timestamp.

### CODE REVIEW

File: src/Matcher/UlidMatcher.php [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+{
+    /**
+     * @var string
+     */
+    public const PATTERN = 'ulid';
+
+    /**
+     * @var string
+     */
+    public const ULID_PATTERN = '[0123456789ABCDEFGHJKMNPQRSTVWXYZabcdefghjkmnpqrstvwxyz]{26}';
```
Comment (norberttech) [inline src/Matcher/UlidMatcher.php]: hey, did you maybe took this pattern from somewhere? If yes, could you please share the source?
Thanks for the contribution!

Comment (WengerK) [reply src/Matcher/UlidMatcher.php]: Hey ! 

Indeed, I've been inspired by the Symfony ULID validation method

https://github.com/symfony/uid/blob/8311a3f6e14c21960e7955452fe52a462d58ad2b/Ulid.php#L41-L52

Comment (norberttech) [reply src/Matcher/UlidMatcher.php]: Would you mind to put that link in a comment to this patrern and mention Symfony uid lib as an inspiration?

**Comment by norberttech** [comment]
Thanks @WengerK !

**Pull request merged**